modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -441,12 +441,60 @@ definition {
 	}
 
 	@description = "Verify that in Update Due Date, the due date is required"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test DueDateIsRequired {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry Title 1");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Workflow.viewTaskAssignedToMyRoles(
+			ddlDataDefinitionName = "Data Definition Name",
+			workflowAssetTitle = "Blogs Entry Title 1",
+			workflowAssetType = "Blogs Entry",
+			workflowTask = "Review");
+
+		Workflow.gotoTaskAssignedToMyRolesActionsCmd(
+			workflowAssetTitle = "Blogs Entry Title 1",
+			workflowTask = "Review");
+
+		var key_workflowAssetTitle = "Blogs Entry Title 1";
+		var key_workflowTask = "Review";
+
+		MenuItem.click(menuItem = "Update Due Date");
+
+		SelectFrame(locator1 = "MyWorkflowTasks#IFRAME_UPDATE_DUE_DATE");
+
+		var currentYear = DateUtil.getCurrentYear();
+		var futureYear = ${currentYear} + 1;
+
+		SelectFrame(locator1 = "WorkflowDefinition#DIALOG_BOX_BODY");
+
+		SelectFrame(locator1 = "IFrame#IFRAME");
+
+		Type(
+			locator1 = "MyWorkflowTasks#DIALOG_BOX_DUE_DATE",
+			value1 = " ");
+
+		Type(
+			locator1 = "MyWorkflowTasks#DIALOG_BOX_DUE_DATE_TIME",
+			value1 = "05:30 PM");
 
-		// TODO LRQA-68861 DueDateIsRequired pending implementation
+		Workflow.confirmWorkflowAction();
```

Comment:
You can create and use the same macro you created at that [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/835/commits/d299c9d40897bf65014e97b95381b3a57e33cbec)